airbyte-integrations/connectors/source-iterable/source_iterable/api.py

```diff
@@ -142,41 +162,49 @@ def path(self, **kwargs) -> str:
 class EmailBounce(IterableExportStream):
     name = "email_bounce"
     data_field = "emailBounce"
+    cursor_field = "createdAt"
 
 
 class EmailClick(IterableExportStream):
     name = "email_click"
     data_field = "emailClick"
+    cursor_field = "createdAt"
 
 
 class EmailComplaint(IterableExportStream):
     name = "email_complaint"
     data_field = "emailComplaint"
+    cursor_field = "createdAt"
 
 
 class EmailOpen(IterableExportStream):
     name = "email_open"
     data_field = "emailOpen"
+    cursor_field = "createdAt"
 
 
 class EmailSend(IterableExportStream):
     name = "email_send"
     data_field = "emailSend"
+    cursor_field = "createdAt"
 
 
 class EmailSendSkip(IterableExportStream):
     name = "email_send_skip"
```

Comment:
we don't need name here and in other places right?

Reply:
It is not entirely true, if we do not fill in the `name` variable explicitly, then this class will respond to the `emailsendskip ` stream, but we are using a different `stream name`.